src/main/java/org/jitsi/jicofo/recording/jibri/BaseJibriRecorder.java

```diff
@@ -41,7 +41,7 @@
  *
  * @author Pawel Domas
  */
-public abstract class CommonJibriStuff
+public abstract class BaseJibriRecorder
```

Comment:
Does this name make sense given it's used for SIP gateway as well?

Reply:
I'm open to better suggestions!